recipes/service-response-messaging.rb

```diff
@@ -0,0 +1,15 @@
+#
+# Cookbook Name:: repose
+# Recipe:: service-response-messaging
+#
+# Copyright (C) 2016 Rackspace Hosting
+#
+
+template "#{node[ 'repose' ][ 'config_directory' ]}/response-messaging.cfg.xml" do
```

Comment:
You might want to clean up any `Space inside square brackets detected.` infractions that rubocop declares.  Consistency ftw.